src/style.css

```
@@ -1,3 +1,80 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.logo img {
+  width: 40px;
+  height: 40px;
+  vertical-align: bottom;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  padding: 0;
   margin: 0;
 }
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child
+{
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  font-size: 12px;
+  height: 60px;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: var(--main-color);
+}
+
+.is-active {
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  background: var(--main-color);
+  border-radius: 8px;
+  width: 100%;
+  height: 4px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.link__1 {
+  color: var(--main-color);
+}
```

we want to link, which is active to have such color, please use selector `.nav__link.is-active` to add this style and also you can do it in the line 59 not to duplikate styles, so it would look like: 
```suggestion
.nav__link:hover,
.nav__link.is-active {
  color: var(--main-color);
}
```